src/BlazorWebView/samples/MauiRazorClassLibrarySample/MauiRazorClassLibrarySample.csproj

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <TargetFramework>$(_MauiDotNetTfm)</TargetFramework>
     <IsPackable>false</IsPackable>
+    <BuildInParallel Condition="'$(Configuration)' == 'Release'">false</BuildInParallel>
```

Comment:
Can we open an issue for the blazor team to investigate?

Reply:
Done - https://github.com/dotnet/maui/issues/11872